packages/snap-store-mobx/src/Search/Stores/SearchResultStore.test.ts

```diff
@@ -251,6 +251,38 @@ describe('SearchResultStore', () => {
 			});
 		});
 
+		it('can configured to preselect certain variants', () => {
```

Comment:
Does this test actually verify that the preselected values are actually selected...?